markdown/faq.md

```diff
@@ -5,48 +5,150 @@ description: THAT Conference's Frequently Asked Questions
 
 Many of you have contacted us and asked questions and many of the questions have been asked a few times. So, here are the questions and our answers. If you have other questions not on the list, please let us know: [Hello@THATConference.com](mailto:Hello@THATConference.com)
 
-**Q: Does THAT Conference focus on a particular technical platform or environment in its sessions?**
 
-&nbsp;&nbsp;&nbsp;**A:**&nbsp; No, we are a technology agnostic, “polyglot” conference. Our focus is the technology community, providing a forum for sharing ideas
-and learning from each other.
+<a href="#section-before" name="section-before">### Before the Conference</a>
 
-<hr/>
 
-**Q: Is THAT Conference family-friendly?**
+#### Counselors
 
-&nbsp;&nbsp;&nbsp;**A:**&nbsp; Yes, we absolutely invite families to stay at the resort during the conference and have created a schedule with a wide line-up
-specifically oriented toward family involvement. This includes learning, exploring, gaming and fun for all ages.
+<a href="#q2" name="q2">**Q: How do I become a Camp Counselor (Speaker)?**</a>
 
-<hr/>
+**A:** We welcome anyone regardless of background, skill level, or age to submit a talk proposal. We have both a professional and family track, where we encourage kids to teach other kids! For more information, please see our [Call for Counselors](https://www.thatconference.com/wi/call-for-counselors) page.
 
-**Q: Does THAT Conference target any specific “age-span” or educational level?**
 
-&nbsp;&nbsp;&nbsp;**A:**&nbsp; While we are “family oriented” and have sessions and events for the entire family, the majority of the sessions are provided by
-technology professionals and are adult-oriented. Anyone interested in technology is invited to attend.
+<a href="#q3" name="q3">**Q: Will I be paid to be a Counselor (Speaker)?**</a>
 
-<hr/>
+**A:** We do not pay any speaker fees to our Counselors. As a selected speaker, our Counselors will receive a full, 4 day ticket, 2 night hotel room (base rate) stay at the Kalahari, all conference provided meals, and of course, some awesome swag!
```

Comment:
be good to add to this about the 3 night max per family here

Reply:
3 night max wording has been added.